datalake/src/main/java/com/sequenceiq/datalake/flow/create/handler/StackCreationHandler.java

```diff
@@ -70,7 +65,6 @@ protected Selectable doAccept(HandlerEvent<StackCreationWaitRequest> handlerEven
             StackV4Response stackV4Response = provisionerService.waitCloudbreakClusterCreation(sdxId, pollingConfig);
             SdxCluster sdxCluster = sdxService.getById(sdxId);
             sdxService.updateRuntimeVersionFromStackResponse(sdxCluster, stackV4Response);
-            setStackCreatedStatus(sdxId);
```

Comment:
why have you removed this?

Reply:
afaik stack created status means it is created in CB, and running state is that everything is finished, currently both happening at the same time, since stack provision is the last step, but later anyone can change it. 
or both have the same message?

Reply:
I removed this because I found this called in two places:

First, in this same method the following occurs:

1. `StackV4Response stackV4Response = provisionerService.waitCloudbreakClusterCreation(sdxId, pollingConfig);`
2. In `waitCloudbreakClusterCreation` the following is called: `sdxStatusService.setStatusForDatalakeAndNotify(DatalakeStatusEnum.STACK_CREATION_FINISHED, "Stack created for Datalake", sdxCluster);`

Second, in the section I removed:

1. `setStackCreatedStatus(sdxId);`
2.`sdxStatusService.setStatusForDatalakeAndNotify(DatalakeStatusEnum.STACK_CREATION_FINISHED, "Datalake stack created", datalakeId);`

Reply:
So it is called twice which is why I removed the call, hopefully that makes sense. This was only made evident because of the duplicate events I saw in the UI.

Reply:
thx for the answer